tests/test_scheduler.py

```diff
@@ -1 +1,12 @@
-# TODO
+from TowerScheduler.scheduler import StateMachine
+from TowerScheduler.scheduler import WakeUp
+
+def test_wakeup_transition():
+    state_machine = StateMachine()
+    wake_up = WakeUp()
```

Comment:
We should just be using a single `WakeUp` object instead of instantiating new ones. I'll add a `get_singleton` kind of method to each state later today, and we can use that here. Assume you can call it here as `WakeUp.get_singleton()`.